docs/components.md

```diff
@@ -12,22 +12,22 @@ These components are considered stable. While in beta, breaking changes may be
-| [simpleprometheus](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/simpleprometheusreceiver) | [resourcedetection](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/resourcedetectionprocessor)       |                                                                                                                      |                                                                                                                              |
-| [smartagent](../internal/receiver/smartagentreceiver)                                                                             | [span](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanprocessor)                                 |                                                                                                                      |                                                                                                                              |
-| [splunk_hec](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/splunkhecreceiver)              |                                                                                                                                             |                                                                                                                      |                                                                                                                              |
-| [zipkin](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/zipkinreceiver)                     |                                                                                                                                             |                                                                                                                      |                                                                                                                              |
-
+| Receivers                                                                                                                         | Processors                                                                                                                            | Exporters                                                                                                            | Extensions                                                                                                                   |
```

Comment:
Seeing some undesired reversions in this file.  You may just want to update from HEAD and add the redis entry again.

Reply:
I am seeing that table contents are in the alphabetical order. So, should we add redis in between the entries or at the end of the table ?

Reply:
Corrected this whole table as before and also added redis entry in alphabetical order.